tilepack/builder.py

```diff
@@ -10,17 +10,18 @@
 import mercantile
 import logging
 import csv
+import gzip
 
 sess = requests.Session()
 adapter = requests.adapters.HTTPAdapter(pool_connections=100, pool_maxsize=200)
 sess.mount('https://', adapter)
 
-# def fetch_tile(x, y, z, layer, format, api_key):
+# def fetch_tile(x, y, z, typ, layer, format, api_key):
 def fetch_tile(format_args):
     sleep_time = 0.5 * random.uniform(1.0, 1.7)
     response_info = []
     while True:
-        url = 'https://tile.mapzen.com/mapzen/vector/v1/{layer}/{zoom}/{x}/{y}.{fmt}?api_key={api_key}'.format(**format_args)
+        url = 'https://tile.mapzen.com/mapzen/{typ}/v1/512/{layer}/{zoom}/{x}/{y}.{fmt}?api_key={api_key}'.format(**format_args)
```

Comment:
Agree. We can default to 512, but it should be settable to other values (like 256).